BuildAutoRest.ps1

```diff
@@ -61,6 +61,7 @@ ForEach ($SDK In $SDKName)
         $OutputFullPath = '{0}/{1}' -f $BaseFolder, [System.String]$Config.'output-folder'
         $ToolsFolderPath = '{0}/Tools' -f $BaseFolder
         $RunPesterTestsFilePath = '{0}/RunPesterTests.ps1' -f $ToolsFolderPath
+        $CleanScript = '{0}/clean.sh' -f $ToolsFolderPath
```

Comment:
When running locally in MacOS, Autorest does not create the bin / obj folders with the correct permissions, the clean.sh script essentially just sets the permission for the bin / obj directories and removes them if they exist.

Reply:
Cant we do this in Powershell?